apps/homeassistant-1/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3'
+
+services:
+  homeassistant-1:
+    image: ghcr.io/home-assistant/home-assistant:2024.2.1
+    container_name: homeassistant-1
+    environment:
+      - TZ=${TZ}
+    restart: unless-stopped
+    ports:
+      - ${APP_PORT}:8209
+    volumes:
+      - /etc/localtime:/etc/localtime:ro
+      - ${APP_DATA_DIR}/data/config:/config
+    networks:
+      - tipi_main_network
+    labels:
+      # Main
+      traefik.enable: true
+      traefik.http.middlewares.homeassistant-web-redirect.redirectscheme.scheme: https
+      traefik.http.services.homeassistant.loadbalancer.server.port: 8123
```

Comment:
Shouldn't be 8209?

Reply:
you are correct. fixed.

Reply:
post changes revoked. jsut the APP_PORT settings is enough. works now